examples/analog_hamiltonian_simulation/04_Running_Analog_Hamiltonian_Simulation_with_local_simulator.ipynb

```diff
@@ -0,0 +1,456 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# show_final_avg_density(result_full)\n",
+    "plot_avg_density_2D(get_avg_density(result_full), register)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0f8623ff",
+   "metadata": {},
+   "source": [
+    "## Run AHS program in the blockade subspace\n",
+    "\n",
+    "As we can see above, it takes a long time to finish the simulation because the *full* Hamiltonian is of size $2^{15}\\times 2^{15}$. However, because of Rydberg blockade, if neighboring atoms are within each other's Rydberg blockade radius $R_b$, they are very unlikely to be excited to the Rydberg states simultaneously. Given that (see [notebook 00](https://github.com/aws/amazon-braket-examples/blob/main/examples/analog_hamiltonian_simulation/00_Introduction_of_Analog_Hamiltonian_Simulation_with_Rydberg_Atoms.ipynb))\n",
+    "\\begin{align}\n",
+    "R_b = \\left[\\frac{C_6}{\\sqrt{\\Delta^2+\\Omega^2}}\\right]^{1/6},\n",
+    "\\end{align}\n",
+    "we have $6.752~ \\mu m<R_b<6.796~ \\mu m$ throughout the program, which is always larger than $6.7~ \\mu m$, the distances between neighboring atoms. Hence we can approximate the full Hamiltonian of the system with a smaller *effective* Hamiltonian. We can take advantage of this fact to speed up the simulation by setting the parameter `blockade_radius` as shown below."
```

Comment:
I think the formatting here is not rendering properly

Reply:
It renders correctly in my browser here: https://github.com/aws/amazon-braket-examples/blob/mao/add_ahs_local_sim/examples/analog_hamiltonian_simulation/04_Running_Analog_Hamiltonian_Simulation_with_local_simulator.ipynb (See attached screen)

<img width="1085" alt="Screen Shot 2022-12-15 at 9 49 36 AM" src="https://user-images.githubusercontent.com/86788920/207891543-3b06b6d1-0c41-4775-9252-27ae69e0f770.png">

Reply:
Ah, but not this part, your are right @math411 .
<img width="1085" alt="Screen Shot 2022-12-15 at 9 51 05 AM" src="https://user-images.githubusercontent.com/86788920/207891803-e215b702-1da5-4776-bb01-093b38959d80.png">

Reply:
Please check. They should look good now.